### DISCUSSION TITLE
Enable Series that have multiple authors

### DISCUSSION THREAD

**Discussion by LokiMidgard** [bookwyrm-social/bookwyrm#3443] [Ideas]
While adding some Books, I stumbled upon a series where every Book has another author.

However when clicking on the series link I only saw one book. While I see why this is implemented in such a way, it will not be able to model such series.

The Series I saw was [Justifiers (German link)](https://www.buechertreff.de/buchreihe/44426-justifiers-markus-heitz-reihenfolge/).

**Comment by mouse-reeve** [comment]
This is related to how series are currently implemented, and would, I believe, be resolved in the same refactor proposed here: https://github.com/bookwyrm-social/bookwyrm/issues/2991
